multi-node/aws/.gitignore

```diff
@@ -1,4 +1,5 @@
 /artifacts/template.json
 /bin
-/clusters
-/cluster.yaml
+/assets
+*.yaml
```

Comment:
The idea is to git-ignore all the random cluster configurations that people will dump in there.